thirdparty/google_analytics/CMakeLists.txt

```diff
@@ -27,6 +27,6 @@ target_link_libraries(google_analytics Qt5::Core Qt5::Network)
 
 if (MSVC)
   set_target_properties( google_analytics PROPERTIES
-    COMPILE_FLAGS "/wd4458"
+    COMPILE_FLAGS "/wd4127 /wd4458 /wd4996"
```

Comment:
Need to add C26439, C26444,C 26452, C26495, C26498, C26812 to the ignore list with Qt 5.15 Beta 1